hs_core/models.py

```diff
@@ -1962,6 +1962,34 @@ def get_irods_storage(self):
         else:
             return IrodsStorage()
 
+    def set_quota_holder(self, setter, new_holder):
```

Comment:
It seems there can be only one quota holder per resource. In that case, there is no need to have a parameter ```setter```.